MAINTAINERS

```diff
@@ -0,0 +1,11 @@
+maintainers:
+- spf13
+- jpmcb
+- jharshman
+- wfernandes
+- eparis
+- anthonyfok
+- bogem
+- bep
+- broady
+emeritus:
```

Comment:
What is `emeritus`? 🤔

Reply:
My understanding is "people who had maintainer/write access at some point but who don't have those permissions anymore". However, in this case we might need both "emeritus" and "inactive", the latter meaning "people who has maintainer/write access but who do not actively merge PRs or reply to issues".

Reply:
Exactly right for "emeritus".
I also think that an "inactive" section may be of value for Cobra.  But wanted to see how the maintainers felt about this whole PR before adding another level.  Thanks for bringing it up though.